src/main/webapp/game.js

```diff
@@ -10,11 +13,52 @@ const CSS_MESSAGE_CLASS_DICT = {
 };
 // TODO: @salilnadkarni, replace with userid from cookie (in datastore)
 const USER_ID = "_" + Math.random().toString(36).substr(2, 9);
+var videoId = "";
+var startTime = 0;
+var endTime = 0;
+
+window.addEventListener('DOMContentLoaded', () => {
+  embedVideo();
```

Comment:
Do you need the embedVideo() to be called twice (upon button click and DOMContent loaded)? If so can you explain why in a comment.

Reply:
Yes, one is to handle refreshes/redirect from lobby page, and one is to handle to new round button being clicked